src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarCloudWebServer.cs

```diff
@@ -118,6 +111,23 @@ public override async Task<IList<SensorCacheEntry>> DownloadCache(ProcessedArgs
             }
         }
 
+        // Do not use the downloaders here, as this is an unauthenticated request
+        public override async Task<Stream> DownloadJreAsync(JreMetadata metadata)
+        {
+            var uri = new Uri(metadata.DownloadUrl);
+            logger.LogDebug(Resources.MSG_JreDownloadUri, uri);
+            return await cacheClient.GetStreamAsync(uri);
```

Comment:
Here, I am re-using the cacheClient since it is a dummy HTTP client.
We can either:
- keep it as-is and rename it to something more generic
- Use two different clients

Any thoughts?